js/components/dataset/resource/form.vue

```diff
@@ -252,7 +252,7 @@ export default {
                 const values = [{id: '', label: ''}].concat(schemas.data);
                 return [{
                     id: 'schema.name',
-                    label: this._('Schema'),
+                    label: this.resource.schema.url ? this._('Schema (Url present)') : this._('Schema'),
```

Comment:
I think having this message with an appropriate wording could do the trick.
How about something more explicit: `Schema (warning: setting it here would override the current schema url value)`?

We should validate with @geoffreyaldebert though.

Reply:
The message is too long and the space in the ui is short. We have no more than a few words